webpack.config.js

```diff
@@ -2,32 +2,114 @@ const defaultWebpackConfig = require('@cerner/webpack-config-terra');
 const HtmlWebpackPlugin = require('html-webpack-plugin');
 const { DefinePlugin } = require('webpack');
 const { merge } = require('webpack-merge');
+const path = require('path');
+const fs = require('fs');
+const TerraDevSite = require('./packages/terra-dev-site/src/webpack/plugin/TerraDevSite');
+
+const html = fs.readFileSync(require.resolve('./tests/terra-dev-site/head.html'), 'utf8');
 
 const terraApplicationConfig = (env = {}) => ({
   entry: {
-    index: './tests/wdio/test-harness/index.jsx',
-  },
-  devServer: {
-    historyApiFallback: true,
-  },
-  output: {
-    publicPath: '/',
+    'terra-application-test/index': './tests/test-harness/index.jsx',
   },
   plugins: [
     new HtmlWebpackPlugin({
       lang: env.defaultLocale || 'en',
-      filename: 'index.html',
-      template: './tests/wdio/test-harness/index.html',
+      filename: 'terra-application-test/index.html',
+      template: './tests/test-harness/index.html',
+      chunks: [
+        'core-js',
+        'terra-application-test/index',
+        'regenerator-runtime',
```

Comment:
This limits the chunks applied to the terra-application test harness to only what it needs.